test/e2e/components/vl-button.js

```diff
@@ -42,13 +44,21 @@ class VlButton extends VlElement {
         return this.hasAttribute('tertiary');
     }
 
-    async isLink() {
-        return this.hasClass('vl-link');
+    async hasIcon() {
+        try {
+            await this.getIcon();
+            return true;
+        } catch(e) {
+            return false;
+        }
```

Comment:
misschien nog beter om de "hasIcon" niet te implementeren in functie van de getIcon zodat je ook daarvoor de dependency op icon niet nodig hebt?

Reply:
De import gebeurt maar als er ook effectief een icon is, dus lijkt me in orde?

Reply:
Offline besproken en het is in orde.